app/_landing_pages/tools.yaml

```diff
@@ -0,0 +1,150 @@
+metadata:
+  title: "Kong Tools"
+  tags:
+    - deck
+    - kic
+    - terraform
+    - konnect-api
+    - admin-api
+content:
+  - row:
+      - column:
+          - type: header
+            config:
+              type: h1
+              text: "Kong Tools"
+              align: center
+
+  - row:
+      - column:
+          - type: structured_text
+            config:
+              header:
+                text: "What are Kong tools?"
+              blocks:
+                - type: text
+                  text: Rate limiting controls the number of API requests a client can make within a specified time frame. 
+                    It's a critical function that helps maintain
+                    the performance and availability of web
+                    services and APIs by controlling the request traffic.
+      - column:
+          - type: mermaid
+            config:
+              alt_text: "A diagram that shows how various Kong tools work, with Konnect as an example"
+              diagram: |
+                graph TD
+                subgraph "Deployment & Configuration"
+                    A[Terraform] -->|Configures| B(Konnect)
+                end
+
+                subgraph "Management"
+                    B -->|Managed by| C(KIC)
+                    B -->|Configures Gateway entities| D(Admin API)
+                end
+
+                subgraph "Configuration Updates"
+                    F(decK) -->|Updates| B
+                    B -->|Updated by| E(Konnect API)
+                end
+
+  - row:
+      - column:
+          - type: structured_text
+            config:
+              header:
+                text: "Which Kong tools should I use?"
+              blocks:
+                - type: text
+                  text: "You can use multiple tools, depending on your use case:"
+                - type: unordered_list
+                  items:
+                    - "**Admin API:** Allows you to configure and manage Kong Gateway entities."
+                    - "**Konnect API:** Allows you to configure and manage Konnect-specific components, like Dev Portal or API Products."
+                    - "**decK:** Allows you to visualize configurations and apply changes in a declarative manner to Kong Gateway or Konnect."
+                    - "**Terraform:** Allows you to manage infrastructure as code and automated deployments to streamline setup and configuration of Konnect and Kong Gateway."
+                    - "**KIC:** Allows you to manage ingress traffic and routing rules."
```

Comment:
I think we talked about not including lists like this (entity landing page was the example?), since all of this information should be on the cards themselves.